styles/go.dev/catppuccin.user.css

```diff
@@ -0,0 +1,897 @@
+          content: url('data:image/svg+xml,@{svg}');
+          filter: none;
+        }
+
+        img.go-Icon.go-Icon--accented[alt='checked']  {
+          @svg: escape('<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="@{accent-color}"><path d="M0 0h24v24H0z" fill="none"/><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2m0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8m-2-5.8-2.6-2.6L6 13l4 4 8-8-1.4-1.4z"/></svg>');
+          content: url('data:image/svg+xml,@{svg}');
+          filter: none;
+        }
+      }
+
+      .UnitMeta-links {
+        img.go-Icon {
+          @svg: escape('<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="@{text}"><path d="M0 0h24v24H0z" fill="none"/><path d="M12 1 3 5v6c0 5.55 3.84 10.74 9 12 5.16-1.26 9-6.45 9-12V5zm0 10.99h7c-.53 4.12-3.28 7.79-7 8.94V12H5V6.3l7-3.11z"/></svg>');
+          content: url('data:image/svg+xml,@{svg}');
+          filter: none;
+        }
+
+        img.depsdev-Icon {
+          @svg: escape('<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="m13.587 16.204 2.217 2.217c.26.253.26.667.007.927l-.008.008L3.342 31.81a.653.653 0 0 1-.92 0L.19 29.58a.653.653 0 0 1 0-.92l12.462-12.462a.655.655 0 0 1 .928-.008c0 .008.007.015.007.015" fill="@{subtext0}"/><circle cx="17" cy="15" r="13" stroke="@{text}" stroke-width="4"/><circle cx="17" cy="15" r="4" stroke="@{text}" stroke-width="4"/></svg>');
+          content: url('data:image/svg+xml,@{svg}');
+        }
+      }
+    }
+    // Search Icon in pkg.go
+    form.Homepage-search--symbol::before {
+      @svg: escape('<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="@{accent-color}"><path d="M0 0h24v24H0z" fill="none"/><path d="m20.49 19-5.73-5.73C15.53 12.2 16 10.91 16 9.5A6.5 6.5 0 1 0 9.5 16c1.41 0 2.7-.47 3.77-1.24L19 20.49zM5 9.5C5 7.01 7.01 5 9.5 5S14 7.01 14 9.5 11.99 14 9.5 14 5 11.99 5 9.5"/></svg>');
+      background-image: url('data:image/svg+xml,@{svg}');
+    }
+    // Deprecated Chip
+    span.Documentation-deprecatedTag {
+      color: @crust;
+    }
+    // Code Snippet Highlights
+    pre {
+      .comment {
+        color: @overlay2;
+      }
+      ins {
+        color: @blue;
+      }
+    }
+  }
+}
+
+@-moz-document url-prefix("https://go.dev/tour") {
+  :root[data-theme="dark"] {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root[data-theme="light"], :root[data-theme="auto"] { // Auto is broken in /tour, defaults to light
```

Comment:
What do you mean about it being broken? Looks like the `data-theme` attribute behaves exactly the same as in the other sections.

Reply:
For some reason, `auto` in https://go.dev/tour/welcome/1 doesn't actually follow `prefers-color-scheme`. Using `prefers-color-scheme: dark` on the unstyled version of the page doesn't actually switch the page to dark-mode. I've made it to default light mode to accommodate this behavior on the page.

Reply:
What do you mean "doesn't actually switch the page to dark-mode"? As long as *we* apply *our* styling correctly it shouldn't matter no?

Reply:
Took another look at this after some needed break. I've aligned it with the other moz-documents and should work all fine and dandy now.

Reply:
Sweet!

Reply:
No need to apologize! I completely understand why. Thanks for thoroughly looking on my 1AM handiwork that really should've been more polished lol.